src/tools/mesh_tester.cpp

```diff
@@ -132,12 +132,14 @@ void Input::parse(int argc, char** argv, CLI::App& app)
   app.add_option("-r,--resolution", resolution,
                  "Resolution of uniform grid. \n"
                  "Set to 1 to run the naive algorithm (without a spatial index). \n"
-                 "Set to less than 1 to use the spatial index with a resolution \n"
-                 "of the cube root of the number of triangles.")
+                 "Set to 2 to use a bounding volume hierarchy spatial index.\n"
```

Comment:
@bmhan12 -- in terms of user interface, I think the choice between the naive `O(n^2)` algorithm and the spatial index should be a separate input parameter, maybe `-m,--method == {naive, spatial_index}`  (or something better?)

Reply:
A separate `method` parameter makes a lot of sense. Overloading the `resolution` parameter as-is makes things unclear.

Reply:
I added a `-m,--method == {"naive", "uniform", "bvh"}` parameter to switch between the algorithms.